app/frontend/schools/coaches/components/SA_Coaches_SchoolIndex.res

```diff
@@ -12,82 +14,163 @@ type formVisible =
 type state = {
   coaches: list<Coach.t>,
   formVisible: formVisible,
+  coachCount: int,
 }
 
 type action =
   | UpdateFormVisible(formVisible)
   | UpdateCoaches(Coach.t)
+  | UpdateCoachCount(bool)
+
+let isSelectedTabArchived = () => {
+  let searchQuery = RescriptReactRouter.useUrl().search
+  searchQuery |> Js.String.includes("archived")
+}
 
 let reducer = (state, action) =>
   switch action {
   | UpdateFormVisible(formVisible) => {...state, formVisible: formVisible}
   | UpdateCoaches(coach) =>
     let newCoachesList = coach |> Coach.updateList(state.coaches)
     {...state, coaches: newCoachesList}
+  | UpdateCoachCount(isArchivedTabSelected) =>
+    let count =
+      List.filter(
+        coach => Coach.exited(coach) === isArchivedTabSelected,
+        state.coaches,
+      )->List.length
+    {...state, coachCount: count}
```

Comment:
I think we should reload on archived and use the coach count.

Reply:
I think, that won't be good for UX. 
What if admin wants to archive multiple coaches, it wouldn't look good when page refreshes on each coach archive.
What you think?

Reply:
I don't think it's worth it, @kaisersakhi. Marking a coach archived/exited is an uncommon action. Tying ourselves / the code in knots to slightly improve the UX on such an action is usually problematic eventually. Complex code is just harder to maintain.

Reply:
Makes sense.